src/shared/common/components/workstack.jsx

```diff
@@ -325,6 +325,7 @@ class WorkstackAllocate extends Component {
             case ColumnRenderer.CASE_LINK:
                 return <td key={row.uuid + column.dataValueKey} className='govuk-table__cell'>
                     <Link to={`/case/${row.caseUUID}/stage/${row.uuid}`} className='govuk-link govuk-!-margin-right-3'>{value}</Link>
+                    <Tags row={row} />
```

Comment:
This could do with a test if possible. Also as mentioned in the hocs-workflow PR would be good if we could improve this with a mapping change and allow for FE class colour. This might need to be done in the server side though...

Reply:
I have added build changes for having a customised way of pulling name and class colour as part of the PR. 
Would you be able to review this again ?